contracts/adder/Cargo.toml

```diff
@@ -5,19 +5,14 @@ authors = ["[your_name] <[your_email]>"]
 edition = "2021"
 
 [dependencies]
-anyhow = "*"
-ink = { version = "4.2.0",  default-features = false }
+ink = { version = "4.2",  default-features = false }
```

Comment:
I'd keep `ink` versions identical in all contracts: e.g. in `access_control` we have `=4.2.1`, which seems legit to me

Reply:
done